cognite/client/_api/datapoints.py

```diff
@@ -419,9 +440,64 @@ def retrieve_dataframe(
                 aggregates=aggregates,
                 granularity=granularity,
                 limit=limit,
-            ).to_pandas()
+            ).to_pandas(include_aggregate_name=include_aggregate_name, complete=complete)
         return pd.concat([id_df, external_id_df], axis="columns")
 
+    def retrieve_dataframe_dict(
```

Comment:
I am leaning towards dropping this method. I really want to try to keep the sdk code base as light as possible, and we have not gotten any explicit feature requests for this. We have the "include_aggregate_name" parameter now, which solves the same problem no? 

I'm not saying that this one method will make the SDK bloated, but we should always be mindful when adding new methods as to what purpose they serve.

We can always create a package where we patch methods in to the SDK. cognite-sdk-plus or something where we can put in all kinds of fancy functionality.

Reply:
the idea would be that you can do things like
quality = df['count'] > 10 & df['totalVariance'] > 0.1
but yes, without feedback it's hard to see who aside from me has use for this

Reply:
Let's keep it, I think it sounds very useful.